public/video-ui/src/services/VideosApi.js

```diff
@@ -121,12 +121,15 @@ export default {
 
     function updateArticleField(stage, data, composerUrl, pageId) {
       if (data.value || data.belongsTo === 'settings') {
+        const value = data.isFreeText
+          ? data.value.split('"').join('\\"')
```

Comment:
Replace only replaces one character so this seemed like the simples way of doing it.